packages/ndla-ui/src/Embed/ConceptEmbed.tsx

```diff
@@ -110,20 +110,22 @@ export const ConceptEmbed = ({ embed, fullWidth, heartButton: HeartButton }: Pro
     return (
       <BlockConcept
         fullWidth={fullWidth}
-        title={concept.title.title}
+        title={concept.title}
```

Comment:
Dette er vel breaking, eller er det kun internt i pakken?

Reply:
Det skal ikke være breaking, da det er kun for articleConverter disse elementene brukes.